src/PluginNodes/AggregateStatePluginNode.cs

```diff
@@ -0,0 +1,114 @@
+﻿namespace OpcPlc.PluginNodes
+{
```

Comment:
Should we rename to ComplexTypeAggregateStatePluginNode.cs for consistency?

Reply:
reanmed